nixos/platform/default.nix

```diff
@@ -20,7 +20,7 @@ let
       )
     else
       [ ];
-  currentPlatform = "24.11";
+  currentPlatform = "25.11";
```

Comment:
Given that I apparently forgot to update this version indentifier in the previous release iteration: Is it really necessary that we hardcode this here, or can this also be inherited from nixpks/NixOS?

Reply:
I don't know, probably this could be inherited